src/ngraph/ngraph_sgcompiler.cc

```diff
@@ -211,15 +213,16 @@ std::shared_ptr<ngraph::Function> SGCompiler::MakeForwardFunction(
   // create the Forward Function object representing the graph
   auto func = std::make_shared<ngraph::Function>(outputs, parameters);
 
-  // fuse conv + bias before autodiff
+// fuse conv + bias before autodiff
+#ifndef MXNET_USE_NGRAPH_IE
```

Comment:
Given that this is inference shouldn't this be turned by IE by default.  I'm wondering if there is a way to condition training behavior to be disabled for IE transformer by default.  It might avoid having macro conditions.  I suppose I can investigate that later 😋

Reply:
Well, soonish, we need to clean up the way fprop cache interacts with fusions, Scott thinks state can do that.

Reply:
@adstraw .  It was nothing too serious.  It's just that IE is inference only, so by design any behavior that involves training shouldn't be apart of the code flow for IE.  Since it is temporary, it should be fine.  I just like to avoid macro conditions as much as possible.  It makes the code less flexible.